alby-nostr-wallet-connect/umbrel-app.yml

```diff
@@ -2,32 +2,32 @@ manifestVersion: 1
 id: alby-nostr-wallet-connect
 name: Nostr Wallet Connect
 tagline: The power of the zaps at the tip of your fingers.
-category: finance
-version: "0.3.0"
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
+category: Finance
+version: "0.4.1"
 port: 58000
 description: >-
-  Connect your LND node over Nostr to zap directly from within supported apps like Damus & Amethyst.
+  Connect your LND node over Nostr.
 
+  Zap directly from within supported apps like Damus & Amethyst.
 
-  🛠️ Instructions
+  Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
 
-  - Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
-
-  - Damus (currently only on Testflight) video guide: https://streamable.com/bareih
+  Damus (currently only on Testflight) video guide: https://streamable.com/bareih
 developer: Alby
 website: http://getalby.com/
 submitter: Alby
-submission: https://github.com/getumbrel/umbrel-apps/pull/636
+submission: https://github.com/getAlby/nostr-wallet-connect
 repo: https://github.com/getAlby/nostr-wallet-connect
 support: https://github.com/getAlby/nostr-wallet-connect/issues
 gallery:
-  - 1.jpg
-  - 2.jpg
-  - 3.jpg
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_1.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_2.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_3.png
```

Comment:
Can you please change these back to the previous 1.jpg, 2.jpg, and 3.jpg that we had last time. These are currently hosted here: https://github.com/getumbrel/umbrel-apps-gallery/tree/master/alby-nostr-wallet-connect

Let me know if you were looking to make changes to these existing gallery images.